### ISSUE TITLE
Support for emscripten?

### ISSUE THREAD

**Issue by paulocoutinhox** [vurtun/nuklear#790]
Hi,

Has support for emscripten?

I want add a gui in my sample for wasm:
https://github.com/ezored/sample-wasm

Thanks.

**Comment by prsolucoes** [comment]
Yes, i know it. But lot of time pass and we don't a clear answer like "Yes, it work. Use this implementation abx.h". I know other projects that use nuklear, but i want know if the things are in the same stage or if it has official support without other repository/project.

**Comment by tanis2000** [comment]
I am using Nuklear with emscripten and it works a charm.

**Comment by dumblob** [comment]
@tanis2000 that's good news - any link to a web app you're testing it with?

**Comment by tanis2000** [comment]
@dumblob my Ludum Dare 43 entry is using Nuklear and is cross-platform, including web with emscripten. You can check out the source code and details here: https://ldjam.com/events/ludum-dare/43/santa-frowns-to-town
